### DISCUSSION TITLE
Currency change

### DISCUSSION THREAD

**Discussion by Valocai** [Paymenter/Paymenter#177] [Ideas]
Built in currency change. So up near your profile you can change the see prices in your own currency.

**Comment by 09han09** [comment]
Implementing a built-in currency change feature can be problematic for several reasons:

1. **Payment Processor Compatibility**: As far as I know, currency conversion can be challenging with different payment gateways like PayPal and Stripe, potentially leading to technical complexities and errors.

2. **Inaccurate Conversion:** Automatically changing prices from one currency to another can sometimes make mistakes in rounding and show prices that don't quite match the real cost. This might confuse users and make them unhappy because they expected different prices. Which leads to the following:

3. **Currency Data Maintenance:** Keeping exchange rates up-to-date is resource-intensive, adding work for developers or requiring businesses to manage fluctuating prices, which might not provide substantial benefits in terms of user convenience.

While it's a decent idea, it can complicate the user experience, create technical challenges, and raise 'privacy and regulatory' concerns, which could outweigh the benefits in some cases.
